code/game/objects/items/devices/PDA/cart.dm

```diff
@@ -374,6 +379,56 @@ Code:
 
 				menu += "</FONT></PRE>"
 
+		if (53)
+			menu = "<h4><img src=pda_alert.png> Please select an Alert Computer</h4><BR>No Alert Computer detected in the vicinity.<BR>"
+			alertmonitor = null
+			alertmonitors = list()
+
+			var/alertcount = 0
+			var/found = 0
+
+			for(var/obj/machinery/computer/station_alert/aMon in machines)
+				if(!(aMon.stat & (NOPOWER|BROKEN)))
+					var/turf/T = get_turf(src)
+					if(T.z == aMon.z)//the application may only detect power monitoring computers on its Z-level.
+						if(!found)
+							menu = "<h4><img src=pda_alert.png> Please select an Alert Computer</h4><BR>"
+							found = 1
+							menu += "<FONT SIZE=-1>"
+						alertcount++
+						menu += "<a href='byond://?src=\ref[src];choice=Alert Select;target=[alertcount]'> [aMon] </a><BR>"
+						alertmonitors += aMon

```

Comment:
To facilitate proper garbage collection i suggest adding the reference in plaintext rather than a hard ref otherwise the GC will have to hard delete it.